### PR TITLE
Fix R1 streams exceeding quota limits

### PR DESCRIPTION
**Pull request by wallyqs**
Fixes #5770 

Signed-off-by: Waldemar Quevedo <wally@nats.io>

### CODE REVIEW

**Review by neilalexander — APPROVED** [review]
LGTM

File: server/jetstream_cluster_3_test.go [hunk 1]
```diff
@@ -6433,3 +6433,98 @@ Consume3:
 		t.Errorf("Consumers to same stream are at different sequences: %d vs %d", a, b)
 	}
 }
+
+func TestJetStreamAccountFileStoreLimits(t *testing.T) {
```
Comment (derekcollison) [inline server/jetstream_cluster_3_test.go:6437]: Naming convention here may mean that Travis will not run this test. We need to check that.

File: server/jetstream_cluster_3_test.go [hunk 2]
```diff
@@ -6433,3 +6433,98 @@ Consume3:
...
+		"R3": {
+			MaxMemory:    1 << 10,
+			MaxStore:     1 << 10,
+			MaxStreams:   -1,
+			MaxConsumers: -1,
+		},
+	}
+
+	// Update the limits in all servers.
+	for _, s := range c.servers {
```
Comment (derekcollison) [inline server/jetstream_cluster_3_test.go:6483]: If you do this by hand just run a normal cluster, no need for separate conf.

**Review by derekcollison — CHANGES_REQUESTED** [review]
Great find, a few comments.

**Comment by wallyqs** [comment]
`TestJWTJetStreamTiers` is failing now so investigating...

**Comment by wallyqs** [comment]
Fixed the test and now green ✅

**Pull request merged**